pkg/kube/client.go

```diff
@@ -449,15 +449,34 @@ func (c *Client) cleanup(newlyCreatedResources []*resource.Info) (cleanupErrors
 //
 // Namespace will set the namespace.
 func (c *Client) Delete(namespace string, reader io.Reader) error {
```

Comment:
Is this function now obsolete?

Reply:
I don't think it is, but even if it is obsolete, we cannot delete it because it would break backwards compatibility (since it is an exported function)

Reply:
Ok, probably best to leave it then.